# BotanyENMWorkshop2021
Using Digitized Collections-Based Data in Research: Applications for Ecology, Phylogenetics, and Biogeography at Botany 2021      
See the html:  https://soltislab.github.io/BotanyENMWorkshops/Demos/Rbased/CrashCourse_2021.html      

**How to cite**: Link the repository in your method section as github.com/soltislab/BotanyENMWorkshops. For example "Models were developed for each species following avaliable scripts (github.com/soltislab/BotanyENMWorkshops)."

## Organizers 
[Shelly Gaynor](https://github.com/mgaynor1), Maria Cortez, [Andre Naranjo](https://github.com/aanaranjo), [Lauren Whitehurst](https://github.com/laurenwhitehurst21), Tal Kinser, Makenzie Mabry, Hector Figueroa, Hannah Marx, Jon Spoelhof, Ryan Folk, CJ Grady, Doug Soltis, and Pam Soltis.     
     
Many people contributted to this material including [Anthony Melton](https://github.com/meltonae), [Johanna Jantzen](https://github.com/jjantzen), and more.  


## Additional Resources   
[iDigBio API Working Group](https://biodiversity-specimen-data.github.io/specimen-data-use-case/)     
[QGIS Introduction - RhettRautsaw/GIS_Tutorial](https://github.com/RhettRautsaw/GIS_Tutorial)           
[SDM Best Practices](https://github.com/plantarum/sdm-best-practices/wiki)    
[mbelitz/Odo_SDM_Rproj](https://github.com/mbelitz/Odo_SDM_Rproj)           
  

## Other related material from our lab past and current members     
[mgaynor1/CURE-FL-Plants](https://github.com/mgaynor1/CURE-FL-Plants)         
[mgaynor1/R4NaturalHistoryCollections-BCEENET2021](https://github.com/mgaynor1/R4NaturalHistoryCollections-BCEENET2021)           
[mgaynor1/long-winded-scripts](https://github.com/mgaynor1/long-winded-scripts)         
[mgaynor1/BLUE-Intro2RwithBiodiversityData](https://github.com/mgaynor1/BLUE-Intro2RwithBiodiversityData)             
[mgaynor1/BCEENET-DataCleaning](https://github.com/mgaynor1/BCEENET-DataCleaning)             
[aemelton/EA_ENA_ENM](https://github.com/aemelton/EA_ENA_ENM)             
[ryanafolk/pno_calc](https://github.com/ryanafolk/pno_calc)             
[ryanafolk/ambitus](https://github.com/ryanafolk/ambitus)           
[ryanafolk/eco-discretizer](https://github.com/ryanafolk/eco-discretizer)           
[richiehodel/Amborella_ENM](https://github.com/richiehodel/Amborella_ENM)         
[jjantzen/CommPhylogeneticsOSBS](https://github.com/jjantzen/CommPhylogeneticsOSBS)       

## Papers to read.        
### Introduction to Natural History Collections    
* Soltis. 2017. Digitization of herbaria enables novel research. American Journal of Botany.     
* Herberling et al. 2019. The changing uses of herbarium data in an era of global change: An overview using automated content analysis. BioScience.      
* Nelson and Ellis. 2018. The history and impact of digitization and digital data mobilization on biodiversity research. Phil. Trans. R. Soc. B.       

### Occurrence Data
* Daru et al. 2017. Widespread sampling biases in herbaria revealed from large-scale digitization. New Phytologist.      
* Zizka et al. 2019. CoordinateCleaner: Standardized cleaning of occurrence records from biological collection databases. Methods in Ecology and Evolution.       
* Aiello-Lammens et al. 2015. spThin: an R package for spatial thinning of species occurrence records for use in ecological niche models. Ecography.              
* Proosdij et al. 2016. Minimum required number of specimen records to develop accurate species distribution models. Ecography.     

       
### Climatic layers      
* Barve et al. 2011. The crucial role of the accessible area in ecological niche modeling and species distribution modeling.  Ecological Modelling.       
* Cobos et al. 2019. An exhaustive analysis of heuristic methods for variable selection in ecological niche modeling and species distribution modeling. Ecological Informatics.        


### ENM methods 
* Peterson. 2001. Predicting species' geographic distributions based on ecological niche modeling. The Condor.    
* Muscarella et al. 2014. ENMeval: An R package for conducting spatially independent evaluations and estimating optimal model complexity for MaxEnt ecological niche models. Methods in Ecology and Evolution.           
* Sillero N. and A. M. Barbosa. 2020. Common mistakes in ecological niche models. International Journal of Geographical Information Science.           
* Jiménez & Soberón. 2020. Leaving the area under the receiving operating characteristic curve behind: An evaluation method for species distribution modelling applications based on presence-only data. Methods in Ecology and Evolution.            
* Cobos et al. 2019. kuenm: an R package for detailed development of ecological niche models using Maxent. PeerJ.           
* Warren et al. 2010. ENMTools: a toolbox for comparative studies of environmental niche models. Ecography.          
* Brown and Carnaval. 2019. A tale of two niche: methods, concepts, and evolution. Frontiers of Biogeography.    
* Warren et al. 2021. The effects of climate change on Australia’s only endemic Pokémon: Measuring bias in species distribution models. Methods in Ecology and Evolution. 
 


### Applications of ENMs
* Allen et al. 2019. Spatial Phylogenetics of Florida Vascular Plants: The Effects of Calibration and Uncertainty on Diversity Estimates. iScience.    
* Marchant et al. 2016. Patterns of abiotic niche shifts in allopolyploids relative to their progenitors. New Phytologist.       
* Gaynor et al. 2018. Climatic niche comparison among ploidal levels in the classic autopolyploid system, Galax urceolata. American Journal of Botany.       
* Visger et al. 2016. Niche divergence between diploid and autotetraploid Tolmiea. American Journal of Botany.         
* Wang et al. 2021. Potential distributional shifts in North America of allelopathic invasive plant species under climate change models. Plant Diversity.       
* Gaynor et al. 2021. Biogeography and ecological niche evolution in Diapensiaceae inferred from phylogenetic analysis. Journal of Systematics and Evolution.       
* Fitzpatrick and Turelli. 2006. The geography of mammalian speciation: Mixed signals from phylogenies and range maps. Evolution.       
* Cardillo and Warren. 2016. Analysing patterns of spatial and niche overlap among species at multiple resolutions. Global Ecology and Biogeography.     
* Jantzen et al. 2019. Effects of taxon sampling and tree reconstruction methods on phylodiversity metrics. Ecology and Evolution.    


### Advanced Topics 
* Webb et al. 2002. Phylogenies and Community Ecology. Annual Reviews of Ecology and Systematics.   
* Lu et al. 2018. Evolutionary history of the angiosperm flora of China. Nature.  
* Pollock et al. 2017. Large conservation gains possible for global biodiversity facets. Nature.  
* Marx et al. 2017. Riders in the sky (islands): Using a mega‐phylogenetic approach to understand plant species distribution and coexistence at the altitudinal limits of angiosperm plant life. Journal of Biogeography.       
* Li et al. 2019. For common community phylogenetic analyses, go ahead and use synthesis phylogenies. Ecology.   
* Tucker et al. 2016. A guide to phylogenetic metrics for conservation, community ecology and macroecology. Biological Reviews.   
* Mishler et al. 2014. Phylogenetic measures of biodiversity and neo- and paleo-endemism in Australian Acacia. Nature Communications.   
* Miller et al. 2016. Phylogenetic community structure metrics and null models: a review with new methods and software. Ecography.    
* Rahbek et al. 2019. Building mountain biodiversity: Geological and evolutionary processes. Science.    
* Körner 2003. Alpine plant life: Functional plant ecology of high mountain ecosystems. Springer. (Book)
* Figueroa et al. 2021. Alpine, but not montane, seed plants constitute a biogeographically and climatically distinct species pool across the Americas. Alpine Botany.   
* Swenson. 2011. The role of evolutionary processes in producing biodiversity patterns, and the interrelationships between taxonomic, functional and phylogenetic biodiversity. American Journal of Botany.    
* Cortez et al. 2021. Is the age of plant communities predicted by the age, stability and soil composition of the underlying landscapes? An investigation of OCBILs. Biological Journal of the Linnean Society.    
* Hopper, Silveira and Fiedler. 2016. Biodiversity hotspots and Ocbil theory. Plant Soil.    
* Schut et al. 2014. Rapid characterisation of vegetation structure to predict refugia and climate change impacts across a global biodiversity hotspot. PLOS ONE.   
* Zappi et al. 2017. Plant biodiversity drivers in Brazilian Campos Rupestres: Insights from phylogenetic structure. Frontiers Plant Science.  
* Linder. 2001. Plant diversity and endemism in sub-Saharan tropical Africa. Journal of Biogeography.    
* Bomblies. 2020. When everything changes at once: finding a new normal after genome duplication. Proceedings of the Royal Society B. 
* Coate and Doyle. 2013. Polyploid and hybrid genomics. (Book)
* Buggs et al. 2011. Transcriptomic shock generates evolutionary novelty in a newly formed, natural allopolyploid plant. Current Biology. 
* McCarthy et al. 2017. Related allopolyploids display distinct floral pigment profiles and transgressive pigments. American Journal of Botany. 
* Van de Peer et al. 2021. Polyploidy: an evolutionary and ecological force in stressful times. The Plant Cell. 
* Schmickl and Yant. 2021. Adaptive introgression: how polyploidy reshapes gene flow landscapes. New Phytologist. 
* Glick and Mayrose. 2014. ChromEvo: Assessing the pattern of chromosome number evolution and the inference of polyploidy along a phylogeny. Molecular Biology and Evolution.    
 




